<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilePicture</class>
 <widget class="QWidget" name="ProfilePicture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lCabecera">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Image" name="imProfile" native="true">
     <zorder>mbBrowse</zorder>
     <zorder>mbBrowse</zorder>
     <zorder>mbBrowse</zorder>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>266</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="MiButton" name="mbBrowse">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MiButton</class>
   <extends>QPushButton</extends>
   <header>view/mibutton.h</header>
  </customwidget>
  <customwidget>
   <class>Image</class>
   <extends>QWidget</extends>
   <header>view/image.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
